### PR TITLE
Add Ortho75

### PR DESCRIPTION
**Pull request by awkannan**
## Description

Adds Ortho75 Keyboard

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/cannonkeys/ortho75/rules.mk [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+# a custom board definition that you plan to reuse).
+# If you're not setting it here, leave it commented out.
+# It is chip dependent, the correct number can be looked up here (page 175):
+# http://www.st.com/web/en/resource/technical/document/application_note/CD00167594.pdf
+# This also requires a patch to chibios:
+#   <tmk_dir>/tmk_core/tool/chibios/ch-bootloader-jump.patch
+#STM32_BOOTLOADER_ADDRESS = 0x1FFFC800
+
+
+#BOOTMAGIC_ENABLE = yes	# Virtual DIP switch configuration
```
Comment (yanfali) [inline keyboards/cannonkeys/ortho75/rules.mk]: we've been recommending BOOTMAGIC=lite to people. Does that work on your keyboards?

Comment (awkannan) [reply keyboards/cannonkeys/ortho75/rules.mk]: I haven't tested it so I'd rather leave it off for now

**Comment by awkannan** [comment]
@drashna added

File: keyboards/cannonkeys/ortho75/info.json [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+        {"label":"6", "x":6, "y":0},
+        {"label":"7", "x":7, "y":0},
+        {"label":"8", "x":8, "y":0},
+        {"label":"9", "x":9, "y":0},
+        {"label":"0", "x":10, "y":0},
+        {"label":"Backspace", "x":11, "y":0},
+        {"label":"0", "x":12, "y":0},
+        {"label":"0", "x":13, "y":0},
+        {"label":"0", "x":14, "y":0},
+        {"label":"0", "x":15, "y":0},
```
Comment (noroadsleft) [inline keyboards/cannonkeys/ortho75/info.json]:
```suggestion
```

File: keyboards/cannonkeys/ortho75/info.json [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+        {"label":"Y", "x":6, "y":1},
+        {"label":"U", "x":7, "y":1},
+        {"label":"I", "x":8, "y":1},
+        {"label":"O", "x":9, "y":1},
+        {"label":"P", "x":10, "y":1},
+        {"label":"Delete", "x":11, "y":1},
+        {"label":"0", "x":12, "y":1},
+        {"label":"0", "x":13, "y":1},
+        {"label":"0", "x":14, "y":1},
+        {"label":"0", "x":15, "y":1},
```
Comment (noroadsleft) [inline keyboards/cannonkeys/ortho75/info.json]:
```suggestion
```

File: keyboards/cannonkeys/ortho75/info.json [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+        {"label":"H", "x":6, "y":2},
+        {"label":"J", "x":7, "y":2},
+        {"label":"K", "x":8, "y":2},
+        {"label":"L", "x":9, "y":2},
+        {"label":";", "x":10, "y":2},
+        {"label":"'", "x":11, "y":2},
+        {"label":"0", "x":12, "y":2},
+        {"label":"0", "x":13, "y":2},
+        {"label":"0", "x":14, "y":2},
+        {"label":"0", "x":15, "y":2},
```
Comment (noroadsleft) [inline keyboards/cannonkeys/ortho75/info.json]:
```suggestion
```

File: keyboards/cannonkeys/ortho75/info.json [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+        {"label":"N", "x":6, "y":3},
+        {"label":"M", "x":7, "y":3},
+        {"label":",", "x":8, "y":3},
+        {"label":".", "x":9, "y":3},
+        {"label":"/", "x":10, "y":3},
+        {"label":"Enter", "x":11, "y":3},
+        {"label":"0", "x":12, "y":3},
+        {"label":"0", "x":13, "y":3},
+        {"label":"0", "x":14, "y":3},
+        {"label":"0", "x":15, "y":3},
```
Comment (noroadsleft) [inline keyboards/cannonkeys/ortho75/info.json]:
```suggestion
```

File: keyboards/cannonkeys/ortho75/info.json [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+        {"label":"Space", "x":5, "y":4},
+        {"label":"Space", "x":6, "y":4},
+        {"label":"Raise", "x":7, "y":4},
+        {"label":"Left", "x":8, "y":4},
+        {"label":"Down", "x":9, "y":4},
+        {"label":"Up", "x":10, "y":4},
+        {"label":"Right", "x":11, "y":4},
+        {"label":"0", "x":12, "y":4},
+        {"label":"0", "x":13, "y":4},
+        {"label":"0", "x":14, "y":4},
```
Comment (noroadsleft) [inline keyboards/cannonkeys/ortho75/info.json]:
```suggestion
        {"label":"0", "x":14, "y":4}
```

File: keyboards/cannonkeys/ortho75/info.json [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+        {"label":"0", "x":15, "y":4}
```
Comment (noroadsleft) [inline keyboards/cannonkeys/ortho75/info.json]:
```suggestion
```

**Comment by awkannan** [comment]
Good catch @noroadsleft ! Thanks :D

File: keyboards/cannonkeys/ortho75/keymaps/default/keymap.c [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
```
Comment (fauxpark) [inline keyboards/cannonkeys/ortho75/keymaps/default/keymap.c]:
```suggestion
```

This doesn't seem to be used.

**Comment by yanfali** [comment]
@drashna unless you have any other objections, this one is probably good to go

File: keyboards/cannonkeys/ortho75/rules.mk (hunk 1 above)
Comment (drashna) [reply keyboards/cannonkeys/ortho75/rules.mk]: In theory, it should.

**Comment by drashna** [comment]
Well, other than that no dfu suffix is set.... But that is wholly optional. 

Looks good, thanks!

**Pull request merged**